docs/install/install-knative-gcp.md

```diff
@@ -60,28 +64,29 @@
         In order to make controller compatible with Workload Identity, use
         [ko](http://github.com/google/ko) to apply
         [controller-gke](../../config/core/deployments/controller-gke.yaml)
-        first.
+        first. It is in `config/core/deployments`.
 
         ```shell
-        ko apply -f controller-gke.yaml
+        ko apply -f config/core/deployments/controller-gke.yaml
         ```
 
         Then you can apply
         [init_control_plane_gke](../../hack/init_control_plane_gke.sh) to
         install all the configuration by running:
```

Comment:
Missing some context here... Do we have need some brief explanation on what the script does?

Reply:
The doc includes two methods to configure it, initialization scripts and manual steps. What the script does is the same as the manual step. In the script, it also has comments for explanation. I feel explanation is redundant here...